src/Riok.Mapperly.Abstractions/MappingConversionType.cs

```diff
@@ -0,0 +1,70 @@
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// A <see cref="MappingConversionType"/> represents a type of conversion
+/// how one type can be converted into another.
+/// </summary>
+[Flags]
+public enum MappingConversionType
+{
+    /// <summary>
+    /// None.
+    /// </summary>
+    None = 0,
+
+    /// <summary>
+    /// Use a target type constructor
+    /// which has a single parameter of the type of the source.
+    /// </summary>
+    Ctor = 1 << 0,
```

Comment:
Would write this out as `Constructor`.

Maybe also change the description to
> Use the constructor of the target type, which accepts the source type as a single parameter.

Reply:
Improved.